test/CLISpec.yml

```diff
@@ -1,35 +1,30 @@
 ---
 - test: Test help output
   steps:
-    -   in: bn --help | sed 's/^.\{2\}//'
+    -   in: bn --help
         out: |-
-
-            Usage: bn [options] [command]
-
-            Binaris command line interface
-
-
-            Options:
-
-              -V, --version      output the version number
-              -p, --path <path>  Use directory dir. "create" will create this directory if needed.
-              -h, --help         output usage information
-
-
-            Commands:
-
-              login                        Login to your Binaris account using an API key
-              logs [options] <function>    Print the logs of a function
-              create [options]             Create a function from template
-              deploy <function>            Deploys a function to the cloud
-              remove <function>            Remove a previously deployed function
-              invoke [options] <function>  Invoke a Binaris function
+              Binaris command line interface
+              
+              Usage: bn <command> [options]
+              
+              Commands:
+                bn create [function] [options]  Create a function from template
+                bn deploy <function> [options]  Deploys a function to the cloud
+                bn remove <function> [options]  Remove a previously deployed function
+                bn invoke <function> [options]  Invoke a Binaris function
+                bn logs <function> [options]    Print the logs of a function
+                bn login                        Login to your Binaris account using an API key
+              
+              Options:
+                --version   Show version number  [boolean]
```

Comment:
> [boolean]

seems a bit out of place. It's a "flag", not a "boolean parameter".. It's not like anyone will do `--version false`...